brainsets/utils/openneuro/openneuro_s3.py

```
@@ -0,0 +1,337 @@
+                        if attempt >= max_attempts:
+                            raise
+                        time.sleep(wait_time)
+                        wait_time = min(wait_time * 2, max_wait)
+
+            return wrapper
+
+        return decorator
+
+    @_retry(max_attempts=5, initial_wait=4, max_wait=10)
+    def _graphql_query(query, variables=None):
+        response = requests.post(
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)